.gitignore

```diff
@@ -1,15 +1,13 @@
 /dist
-gen.go
+/autogen/gen.go
```

Comment:
I'd love to get rid of the extra generator step. I haven't worked a lot with generated code, however, so no idea if checking that in is a good practice or not.

(I know the Kubernetes folks are trying hard to undo committing generated code, but that might be because they are playing in a different league.)

Either way, discussing / considering in a separate issue / PR sounds good. 👍